### PR TITLE
added getsolvetime

### PR DESCRIPTION
**Pull request by emreyamangil**
added getsolvetime

### CODE REVIEW

**Comment by emreyamangil** [comment]
wops! I broke the constructor, fixed it and added a test, trying to figure out how to update this pull request :)

File: src/CplexSolverInterface.jl [hunk 1]
```diff
@@ -8,6 +8,7 @@ type CplexMathProgModel <: AbstractMathProgModel
     branchcb
     incumbentcb
     infocb
+    solvetime
```
Comment (joehuchette) [inline src/CplexSolverInterface.jl]: This should probably be
```jl
solve time::Float64
```
(the constructor will have to be updated below, probably initialized to NaN)

**Comment by emreyamangil** [comment]
Done!!

**Comment by joehuchette** [comment]
LGTM

**Pull request merged**